src/main/java/model/converter/PostfixConverter.java

```diff
@@ -7,45 +7,40 @@
 import java.util.List;
 import java.util.Stack;
 
+import static constant.Operator.*;
+
 public class PostfixConverter implements Converter {
     private static final String WHITESPACE = " ";
+    private static final Stack<Operator> operatorStack = new Stack<>();
 
     @Override
-    public List<String> covert(Expression expression) {
+    public List<String> convert(Expression expression) {
         List<String> postfixList = new ArrayList<>();
-        Stack<Operator> operatorStack = new Stack<>();
-        String[] noWhitespaceExpression = expression.getExpression().split(WHITESPACE);
-
-        for (String textSegment : noWhitespaceExpression) {
-            separateNumberAndOperator(postfixList, operatorStack, textSegment);
+        String[] nonSpaceExpression = convertToNonSpaceExpression(expression);
+
+        for (String textSegment : nonSpaceExpression) {
+            if (isOperator(textSegment)) {
+                Operator operator = Operator.findOperator(textSegment);
+                processLowPriorityOperators(postfixList, operator);
+                operatorStack.push(operator);
+                continue;
+            }
+            postfixList.add(textSegment);
```

Comment:
continue를 쓰는 것 보다는 isOperand()를 구현하는 건 어떨까요?
```suggestion
            if (isOperator(textSegment)) {
                Operator operator = Operator.findOperator(textSegment);
                processLowPriorityOperators(postfixList, operator);
                operatorStack.push(operator);
                continue;
            }
            if (isOperand(textSegment)) {
                 postfixList.add(textSegment);
            }
```

Reply:
`else`와 `continue`을 어떻게 지양해야 하는지 어려웠는데 알려주셔서 감사합니다!🙇‍♂️🙇‍♂️

### 변경
```java
    private boolean isOperand(String textSegment) {
        return textSegment.matches(IS_NUMBER_PATTERN);
    }
```